src/main/java/mate/academy/spring/model/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,14 @@
 package mate.academy.spring.model.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 public class MovieRequestDto {
+    @NotNull
+    @Size(min = 5, message = "Size title should be over 5")
     private String title;
```

Comment:
I guess min = 1 will be more appropriate)
![Screenshot 2022-07-08 at 08 30 06](https://user-images.githubusercontent.com/60258373/177923401-cf95bead-8809-4151-87d4-6005219512a5.png)